addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -42,6 +45,10 @@
     private final Logger logger = LoggerFactory.getLogger(KodiConnection.class);
 
     private static final int VOLUMESTEP = 10;
+    // possible are: int speeds[] = new int[] { -32, -16, -8, -4, -2, -1, 0, 1, 2, 4, 8, 16, 32 };
```

Comment:
should we just make a note about the extra values instead of duplicating everything